frontend/src/components/error/ErrorBoundary.tsx

```diff
@@ -40,9 +41,24 @@ class ErrorBoundary extends React.Component<ErrorBoundaryProps, ErrorBoundarySta
       const { error, errorInfo } = this.state;
       return (
         <div className="pf-v5-u-p-lg">
-          <Title headingLevel="h1" className="pf-v5-u-mb-lg">
-            An error occurred.
-          </Title>
+          <Split>
+            <SplitItem isFilled>
+              <Title headingLevel="h1" className="pf-v5-u-mb-lg">
+                An error occurred.
+              </Title>
+            </SplitItem>
+            <SplitItem>
+              <Button
+                variant="plain"
+                aria-label="Close"
```

Comment:
Is clicking the "close" button supposed to get me out of the error screen and into the view that this was blocking? Just checking coz I don't see that happening on my end. The error that I caused was a misspelled component.

Reply:
@manaswinidas if the error reoccurs, then the error screen will be shown again. There's no way out of that other than to change the URL. Perhaps that's useful to supply a button to go back to the home page?

If however the error occurs because the user interacted with something on the page that caused the error, then the page will refresh without the error unless they perform the same action again.

Reply:
I see. Any scenarios where I can see the close button in action? I'm not sure how to test the close button.

Reply:
Unless we have a known bug in the app, i don't know how you can reproduce this without doing a debugging hack as mentioned in the description.